nginx/server.conf

```diff
@@ -0,0 +1,33 @@
+server {
+  listen 80;
+  listen [::]:80;
+
+  server_name ${NGINX_HOST};
+
+  return 301 https://${NGINX_HOST}$request_uri;
+}
+
+server {
+  listen 443 ssl http2;
+  listen [::]:443 ssl http2;
+
+  server_name ${NGINX_HOST};
+
+  ssl_certificate     /certs/fullchain.pem;
+  ssl_certificate_key /certs/privkey.pem;
+  ssl_prefer_server_ciphers on;
+  ssl_ciphers EECDH+CHACHA20:EECDH+AES128:RSA+AES128:EECDH+AES256:RSA+AES256:EECDH+3DES:RSA+3DES:!MD5;
+
+  access_log /var/log/nginx/access.log;
+  error_log /var/log/nginx/error.log;
```

Comment:
Quick question, we run a single nginx container. When we do `docker logs` on nginx, will we see anything? Because we log to these files here. Should we log to stdout / stderr in the nginx container, and then `docker logs` will show the nginx logs?

Reply:
Yes, requests and fails are currently output via `docker log`. I can look if settings these to stderr will give us more information

Reply:
Did some research. This is actually done by the [official nginx docker images](https://github.com/nginxinc/docker-nginx/blob/2ef3fa66f2a434cd5e44e35a02f4ac502cf50808/mainline/buster/Dockerfile#L96), no need for changes.